src/models/track/Track.jsx

```diff
@@ -35,7 +36,7 @@ function Train({ args = [38, 8, 10], position = [-145.84, 3.42, 54.67], rotation
         <mesh geometry={n.train_7.geometry} material={m.steelClone} {...config} />
         <mesh geometry={n.train_8.geometry} material={m.lightRedClone} {...config} />
         <mesh geometry={n.train_9.geometry} material={m.darkClone} {...config} />
-        {ready && <PositionalAudio url="/sounds/train.mp3" loop autoplay distance={5} />}
+        {sfx && ready && <PositionalAudio url="/sounds/train.mp3" loop autoplay distance={5} />}
```

Comment:
I'm sure there must be a way to mute all audio without using that `sfx` prop as a conditional render

something like this in an effect:
`document.querySelectorAll("audio").forEach( elem => muteAsset(elem) )`

Reply:
ah good point let me try

Reply:
that would be quite dirty though grabbing into the dom like that, and it will cause syncing issues. in that case i'd prefer an audio controller driven by context or something like that.

Reply:
Yes I agree, an audio controller with context is the way to go. I just don't like the way it currently reads and updating will be a pain, but 3-4 sounds is manageable this way

Reply:
ok, let's keep it in mind as sounds get more complex. i also really dislike that brain dead checking for isPlaying before starting or else you get a dom exception. i think there are a lot of benefits using a dedicated controller. but doing that later is 👍